index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title></title>
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
+	<link rel="stylesheet" href="quiz.css">
+</head>
+<body>
+<div class="container-fluid">
+	<nav class="navbar navbar-inverse">
+  	<input type="text" class="input form-control" placeholder="click on car to edit decription.">
+  	<button id= "clear" class="btn btn-default">Clear Description</button>
+	</nav>
+	<div class="row">
+		
+	</div>
+</div>
+<script src="loader.js" type="text/javascript" charset="utf-8" async defer></script>
+<script src="clickhandlers.js" type="text/javascript" charset="utf-8" async defer></script>
+<script src="activeEvents.js" type="text/javascript" charset="utf-8" async defer></script>
+<script type="text/javascript" src="clickhandlers.js"></script>	
```

Comment:
You're pulling in this file twice, which is causing the code to throw errors. In this state none of the JS loads or runs.